Python/lock.c

```diff
@@ -47,18 +47,19 @@ _Py_yield(void)
 #endif
 }
 
+#undef PyMutex_Lock
 void
-_PyMutex_LockSlow(PyMutex *m)
+PyMutex_Lock(PyMutex *m)
```

Comment:
You might move it to the end to still use the static inline macro in the following lines.